src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  font-family: Roboto, serif;
+  background-color: aqua;
+  margin: 0;
+  padding: 0;
+
+  --main-color: #00ACDC;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 60px;
+  padding: 0 50px;
+}
+
+ul.nav_list li {
+  display: inline-flex;
+}
+
+.nav_list {
+  line-height: 60px;
+  font-weight: bold;
+  font-size: 12px;
+  text-transform: uppercase;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  color: black;
+  height: 60px;
+  text-decoration: none;
+  margin-left: 10px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children